tests/__init__.py

```diff
@@ -11,9 +12,9 @@ class DefaultParser(Parser):
     grammar = jexl_grammar(default_config)
 
     def __init__(self, config=None):
-        super().__init__(config or default_config)
+        super(DefaultParser, self).__init__(config or default_config)
```

Comment:
Looks like future also supports [importing a super that works like in Python 3](http://python-future.org/imports.html#explicit-imports).